subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_aux.c

```diff
@@ -39,6 +39,11 @@
 #include "common/log.h"
 #include "hal/debug.h"
 
+static uint8_t lll_adv_connect_rsp_pdu[PDU_AC_LL_HEADER_SIZE +
```

Comment:
I can use scratch buffer, but since RX PDU is already there I need to allocate rx node and copy it there before it can be used for TX. This means critical ISR code is longer. Having this in separate buffer also allows to optimize by initializing PDU only once since we only need to update AdvA and TargetA for each connection.